gettsim/benefits/kiz.py

```diff
@@ -17,7 +15,7 @@ def kiz(household, tb):
         This is done by some fixed share which is updated on annual basis
         ('jährlicher Existenzminimumsbericht')
     """
-    household["uhv_tu"] = aggr(household, "uhv", "all_tu")
+    household["uhv_tu"] = household.groupby(["tu_id"])["uhv"].transform("sum")
```

Comment:
no need for squared brackets in groupby.

Reply:
Fixed